.github/workflows/automatic_pull_request_review.yml

```diff
@@ -0,0 +1,52 @@
+name: Automatic pull request review
+
+on:
+  pull_request:
+    types: [opened, reopened, edited, synchronize]
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Run SwiftLint
+      uses: norio-nomura/action-swiftlint@3.1.0
+      with:
+        args: --strict
+
+  review_pull_request:
+    name: Pull request review
+    runs-on: macOS-latest
+    steps:
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.5.0
+      with:
+        access_token: ${{ github.token }}
+
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - uses: actions/cache@v3
+      id: bunlderCache
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gems-
+
+    - name: Bundle install
+      run: bundle install --path vendor/bundle
```

Comment:
Other workflows have this `bundle install` running step but without the `--path vendor/bundle`, kindly update them too 🙏  And don't forget to add `vendor/bundle` to `gitignore` since we occasionally want to run `bundle install` on our local machine, at least when first checking out.